### ISSUE TITLE
core dumped (version 0.2.0.2)

### ISSUE THREAD

**Issue by jq8778** [uPlexa/uplexa#7]
system: ubuntu 16.04 LTS (under a docker image)

Linux bec29badd505 4.15.0-29-generic #31-Ubuntu SMP Tue Jul 17 15:39:52 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

logs:

2019-02-01 12:13:58,443 INFO  [default] Page size: 4096
Illegal instruction (core dumped)

version:0.2.0.2

The system can run other daemon without problems(have tried trtl)

**Comment by uPlexa** [comment]
Hi there, unfortunately that does not give us much to work with. Could you provide some gdb logs?
https://www.gnu.org/software/gdb/

Thanks!

**Comment by jq8778** [comment]
GNU gdb (Ubuntu 7.11.1-0ubuntu1~16.5) 7.11.1
Copyright (C) 2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.  Type "show copying"
and "show warranty" for details.
This GDB was configured as "x86_64-linux-gnu".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<http://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
<http://www.gnu.org/software/gdb/documentation/>.
For help, type "help".
Type "apropos word" to search for commands related to "word"...
Reading symbols from ./uplexad...done.
(gdb) run
Starting program: /root/daemon/upx_test/uplexa-command-linux64/uplexad
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
2019-02-01 18:37:28,471 INFO  [default] Page size: 4096

Program received signal SIGILL, Illegal instruction.
0x00005555556ed7fe in std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(char const*, std::allocator<char> const&) [clone .isra.66] ()

**Comment by jq8778** [comment]
BTW, tried to build from source and it worked... this problem occur when i tried to use the release built in github...

**Comment by cryptomaxsun** [comment]
Similar problem.
ubuntu~16.04.10
```
./uplexad
2019-02-05 13:54:58,206 INFO  [default] Page size: 4096
Illegal instruction
```

**Comment by uPlexa** [comment]
Will be patched in 2.1.0
